app/views/examples/pass-data/vehicle-check-answers.html

```diff
@@ -0,0 +1,88 @@
+{% extends "layout.html" %}
+
+{% block page_title %}
+  Check your answers
+{% endblock %}
+
+{% block content %}
+
+<main id="content" role="main">
+  
+  <div class="grid-row">
+    <div class="column-two-thirds">
+      <h1 class="heading-large">
+        Check your answers before sending your application
+      </h1>
+
+      <p>
+        When a user has entered data on pages, show it by using code like this:  
+      </p>
+
+      <pre>
+        <div class="code">{% raw %}
+  &lt;tr&gt;
+    &lt;td&gt;
+      Vehicle type:
+    &lt;/td&gt;
+    &lt;td&gt;
+      {{vehicleType}}
+    &lt;/td&gt;
+  &lt;/tr&gt;
+  {% endraw %}</div></pre>
+    </div>
+  </div>
+
+  <div class="form-group">
+
+    <table class="check-your-answers">
+
+      <thead>
+        <tr>
+          <th colspan="2">
+            <h2 class="heading-medium">
+              Vehicle details
+            </h2>
+          </th>
+        </tr>
+      </thead>
+
+      <tbody>
+        <tr>
+          <td>
+            Registration number
+          </td>
+          <td>
+            {{vehicleRegistration}}
+          </td>
+        </tr>
+        <tr>
+          <td>
+            Vehicle type
+          </td>
+          <td>
+            {{vehicleType}}
+          </td>
+        </tr>
+        <tr>
+          <td>
+            Vehicle features
+          </td>
+          <td>
+            {{vehicleFeatures}}
```

Comment:
If you have more than one checkbox selected, they are output without a space between them.